static/docs/commands-reference/cache_dir.md

```diff
@@ -65,4 +65,10 @@ $ cat .dvc/config
 ...
 ```
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
```

Comment:
not need to put this note here, let's not complicate it

Reply:
removed